### PR TITLE
Rework topic on notifications

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Rework topic on notifications

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [x] 21.10.x (master)

### CODE REVIEW

**Review by tuntoja — APPROVED** [review]

File: fr/alerts-notifications/notif-configuration.md [hunk 1]
```diff
@@ -37,110 +142,40 @@ contacts et groupes de contacts qui seront notifiés sont disponibles :
...
+
+- Les notifications sont-elles activées pour le contact ? Les notifications sont-elles configurées correctement sur le contact ?
+
+- Les notifications sont-elles configurées correctement sur l'hôte ou le service ?
+
+### Mes notifications n'arrivent pas suivant l'intervalle spécifié
+
+Vérifiez que la valeur que vous avez définie dans **Intervalle de notification** (dans le 2è onglet) est bien un multiple de la valeur définie dans **Intervalle normal de contrôle** (dans le 1er onglet).
+
+En effet, une notification ne peut être envoyé que suite à un contrôle. Par exemple, si vous décidez que les contrôles ont lieu toutes les heures mais que les notifications doivent être envoyées toutes les 10 minutes, les notifications seront en réalité envoyées toutes les heures, car les contrôles ont lieu toutes les heures et non toutes les 10 minutes.
```
Comment (thiuyendang) [inline fr/alerts-notifications/notif-configuration.md:173]:
```suggestion
En effet, une notification ne peut être envoyée que suite à un contrôle. Par exemple, si vous décidez que les contrôles ont lieu toutes les heures mais que les notifications doivent être envoyées toutes les 10 minutes, les notifications seront en réalité envoyées toutes les heures, car les contrôles ont lieu toutes les heures et non toutes les 10 minutes. 
```

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Review by thiuyendang — APPROVED** [review]

File: en/alerts-notifications/notif-configuration.md [hunk 2]
```diff
@@ -35,104 +142,35 @@ and groups of contacts that will be notified are available:
...
 
-Go to the `Administration > Parameters > Centreon UI` menu and select
-the desired option:
+> The **Cumulative inheritance** option is the easiest to program.
 
-![image](../assets/alerts/notif_centreon_config.png)
+## Troubleshooting
 
-Then click on the **Save** button.
+### Contacts are not receiving notifications
```
Comment (omercier) [inline en/alerts-notifications/notif-configuration.md:149]:
```suggestion
### Contacts are not receiving notifications e-mails
```

Comment (omercier) [reply en/alerts-notifications/notif-configuration.md:149]: This troubleshooting hint applies only if the notification channel is `e-mail`.

**Review by omercier — CHANGES_REQUESTED** [review]

File: en/alerts-notifications/notif-configuration.md [hunk 3]
```diff
@@ -1,31 +1,138 @@
...
+4. [Deploy](../monitoring/monitoring-servers/deploying-a-configuration.html) the configuration.
+
+
+### Step 3: Enabling notifications for the contacts you have defined
+
+1. Go to **Configuration > Users > Contacts/Users** and then click on the contact you want to be notified.
+
+2. On the **General Information** tab, in section **Notification**, check that **Enable notifications** is set to **Yes**. 
+    
+    If the option is set to **Default**, Centreon will use the value defined on the closest parent template. If no value is defined on any parent template, **Default** means **No**.
```
[the remark this answers is not in the document]
Comment (cg-tw) [reply en/alerts-notifications/notif-configuration.md:89]: It depends on whether the contact has been configured to receive notifications (at host level)

**Review by omercier — CHANGES_REQUESTED** [review]
One last thing ;-)

**Review by omercier — APPROVED** [review]

**Pull request merged**